api/fixtures/example.go

```diff
@@ -416,6 +416,8 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 			},
 		}
 		nodePools = append(nodePools, nodePool)
+	default:
```

Comment:
How about an explicit check for the None platform, and make the default cause return an unsupported platform error?

Reply:
That make sense to me, at least for now. See for more discussion and context https://coreos.slack.com/archives/C01C8502FMM/p1643634981500229?thread_ts=1643634480.443369&cid=C01C8502FMM

Reply:
Changed the code to have case for `None` and `Agent` platforms
Default call to panic (like in another switch case in this function)